lib/index.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+/**
+ * Module Dependencies
+ */
+
+var integration = require('@segment/analytics.js-integration');
+var each = require('@ndhoule/each');
+var foldl = require('@ndhoule/foldl');
+var qs = require('qs');
+
+/**
+ * Expose `DoubleClick Floodlight` integration.
+ */
+
+var Floodlight = module.exports = integration('DoubleClick Floodlight')
+  .option('source', '')
+  .tag('standard', '<img src="http://ad.doubleclick.net/activity;src={{ src }};type={{ type }};cat={{ cat }};ord={{ ord }}?">')
+  .tag('custom', '<img src="http://ad.doubleclick.net/activity;src={{ src }};type={{ type }};cat={{ cat }};ord={{ ord }};{{ customVariables }}?">')
```

Comment:
Instead of having two tags here, we could just have the `custom` one (probably renamed), and then have `customVariables` be prepended with a `;` if its nonempty